packages/cli/src/environments.ee/source-control/source-control.service.ee.ts

```diff
@@ -772,6 +777,7 @@ export class SourceControlService {
 				conflict: true,
 				file: item.filename,
 				updatedAt: new Date().toISOString(),
+				owner: item.ownedBy && typeof item.ownedBy === 'object' ? item.ownedBy : undefined,
```

Comment:
I'm not sure I see any place in the codebase where ownedBy could be a string :thinking:

Reply:
It seems to be that if it is a string it is the same as the email of a personal project. Based on the (`findOrCreateOwnerProject`) function in the SourceControlImportService. I guess this is backwards compatible with some old git format. But I'll move it to the same behaviour here as well. 👍